aws/rapids_sagemaker_hpo.ipynb

```diff
@@ -642,7 +642,7 @@
    "metadata": {},
```

Comment:
<p>We can comment out this whole cell and the next one, basically eliminating building another image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>